examples/drivers/windows/otLwf/address.c

```diff
@@ -450,8 +450,8 @@ otLwfTunAddressesUpdated(
                 "6CLL", 
                 &pAddr, 
                 &addr.mPrefixLength, 
-                &addr.mValidLifetime, 
-                &addr.mPreferredLifetime);
+                &addr.mValid,
+                &addr.mPreferred);
```

Comment:
I think we need to use a local (`uint32_t`) variable here to decode the spinel command. Then copy the value to `mValid` & `mPreferred` as a `bool`.